### ISSUE TITLE
Add your projects in the projects folder

### ISSUE THREAD

**Issue by Mustafiz04** [Mustafiz04/Contribute-to-HacktoberFest2020#6]
### **If you have made a nice project such as a web app, android app, simple python projects, etc. Add them to the Projects Folder with proper naming.**

``KEEP CONTRIBUTING``

**Comment by AnubhavHawk** [comment]
Are static HTML pages with nice styling allowed?

**Comment by Lakshitk** [comment]
Hey, can I be assigned this, I have a project that I can add to the folder as required

**Comment by Mustafiz04** [comment]
> Are static HTML pages with nice styling allowed?

Yes, you can.

**Comment by Mustafiz04** [comment]
> Hey, can I be assigned this, I have a project that I can add to the folder as required

Yes, you can contribute your project here.
@Lakshitk go for it.

**Comment by Glitchy-prog** [comment]
I am new to git hub can anyone help me on how to delete a pull request

**Comment by AnubhavHawk** [comment]
Hey, according to the hacktoberfest page, the label should be "hacktoberfest" and not "hacktoberfest2020" will be any issue (just curious)?

**Comment by dhruvinfo28** [comment]
I'll like to add some algorithms related to graphs in the data structure graphs folder.

**Comment by Mustafiz04** [comment]
> I'll like to add some algorithms related to graphs in the data structure graphs folder.

Yes, you can. go for it.

**Comment by Glitchy-prog** [comment]
Where should I add a turtle python animations 
should it be in the projects folder under a new folder

**Comment by AnubhavHawk** [comment]
Yes, Inside projects directory with your project name :)

**Comment by AnubhavHawk** [comment]
Inside [Algorithms](https://github.com/Mustafiz04/Contribute-to-HacktoberFest2020/tree/main/Algorithms) directory. If the algo is of specific type, (say *Backtracking* or *Dynamic Programming* or other ) then place it in the corresponding folder.
Otherwise just put in the [Algorithms](https://github.com/Mustafiz04/Contribute-to-HacktoberFest2020/tree/main/Algorithms)

**Comment by petulla28** [comment]
Ok thanks !

**Comment by Rebaiahmed** [comment]
Hi , I want to add deployed react app (todo-list-app ) and angularjs project deployed to firebase , and another machine learning project , i would to check if this repo is actively now :)

**Comment by Mustafiz04** [comment]
@Rebaiahmed yes this repo is active.

**Comment by ishika1501** [comment]
Hey , I  have a latest corona  update project with full animation and mapping system . I would like to contribute , please assign it to me.

**Comment by Archie200** [comment]
I want to add my project .Please assign me this issue. @Mustafiz04

**Comment by Nimo160302** [comment]
Yes I made a project on Web Scraping,  can i add it here also i want some help in it regarding openpyxl library in it. @Mustafiz04
